code/5905-mario-krstevski/script.js

```diff
@@ -1,3 +1,120 @@
+    { rank: 6, name: "Dalek", series: 1, era: "Modern", broadcastYear: 2005, director:"David Evans", writer:"Robert Shearman", doctor:"Christopher Eccleston (9th Doctor)", companion:"Billie Piper (Rose Tyler)", castCount:"6", plot:"The Doctor encounters a Dalek." },
+    { rank: 7, name:"The Long Game", series:"1", era:"Modern", broadcastYear:"2005", director:"Brian Grant", writer:"Russell T Davies and others", doctor:"Christopher Eccleston (9th Doctor)", companion:"Billie Piper (Rose Tyler)", castCount:"6", plot:"The Doctor and Rose travel to the future." },
+    { rank :8 , name : 'Father\'s Day', series : '1', era : 'Modern', broadcastYear : '2006', director : 'Joe Ahearne', writer : 'Paul Cornell', doctor : 'David Tennant (10th Doctor)', companion : 'Billie Piper (Rose Tyler)', castCount : '7', plot : 'Rose tries to save her father.' },
+    { rank :9 , name : 'The Empty Child', series : '1', era : 'Modern', broadcastYear : '2006', director : 'James Hawes', writer : 'Steven Moffat', doctor : 'David Tennant (10th Doctor)', companion : 'Billie Piper (Rose Tyler)', castCount : '8', plot : 'A child in wartime London.' },
+    { rank :10 , name : 'The Parting of the Ways', series : '1', era : 'Modern', broadcastYear : '2005', director : 'Joe Ahearne', writer : 'Russell T Davies', doctor : 'Christopher Eccleston (9th Doctor)', companion : 'Billie Piper (Rose Tyler)', castCount : '8', plot : 'The Ninth Doctor faces his greatest challenge.' },
+    { rank: 11, name:"The Time Warrior", series:"11", era:"Classic", broadcastYear:"1973", director:"Alan Bromly", writer:"Robert Holmes", doctor:"Jon Pertwee (3rd Doctor)", companion:"Elisabeth Sladen (Sarah Jane Smith)", castCount:"8", plot:"The Doctor battles a Sontaran." },
+    { rank: 12, name:"Genesis of the Daleks", series:"12", era:"Classic", broadcastYear:"1975", director:"David Maloney", writer:"Terry Nation", doctor:"Tom Baker (4th Doctor)", companion:"Elizabeth Sladen (Sarah Jane Smith)", castCount:"6", plot:"The origins of the Daleks are revealed." },
+    { rank: 13, name:"City of Death", series:"17", era:"Classic", broadcastYear:"1979", director:"Douglas Adams & Michael Hayes ", writer:"Douglas Adams & David Fisher ", doctor:"Tom Baker (4th Doctor)", companion:"Lalla Ward (Romana II)", castCount:"7", plot:"A time-traveling adventure in Paris." },
+    { rank: 14, name:"Remembrance of the Daleks", series:"25", era:"Classic", broadcastYear:"1988", director:"Andrew Morgan ", writer:"Eric Saward ", doctor:"Sylvester McCoy (7th Doctor)", companion:"Sophie Aldred (Ace)", castCount:"10 ", plot :"The Daleks return to London." },
+    { rank: 15, name :"Blink" , series :"3" , era :"Modern" , broadcastYear :"2007" , director :"Hettie Macdonald" , writer :"Steven Moffat" , doctor :"David Tennant (10th Doctor)" , companion :"Catherine Tate (Donna Noble)" , castCount :"6" , plot :"Timey-wimey adventures with Weeping Angels." }
+];
+
+const doctors = [
```

Comment:
The list of doctors should be extracted from the data (just like the companions are)